### DISCUSSION TITLE
Keyball44 左手側 左上のキーだけが反応しない

### DISCUSSION THREAD

**Discussion by Leko** [Yowkees/keyball#305] [Q&A] [answered]
![PXL_20230813_034851832](https://github.com/Yowkees/keyball/assets/1424963/71c8b865-a882-4a3d-8ff8-e07fefde5b63)
<img width="1002" alt="image" src="https://github.com/Yowkees/keyball/assets/1424963/62231c2e-6828-489a-9921-1db6b2985339">

添付画像の右上のキーだけが反応しなくなりました。それ以外のキーは全て反応しております。
キースイッチを交換する前までは反応していたのですが、キースイッチを交換する際に力んでしまったのが原因なのか、それから反応しなくなりました。
確認にはRemapのTest Matrix機能を利用しております。

キーソケットを外して別のソケットをはんだ付けし直したのですが、それでも動きませんでした。
はんだも数回付け直しているので、ソケットのはんだ不良ではないと思います。

ダイオードのはんだ付けを見直したいのですが、このキーと繋がっているダイオードはどれになりますか？
また他の原因として考えられることはございますか？

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

１キーのみ不調ということで、問題のキー周辺に不具合があると思われます。
該当のダイオードは下記添付画像の赤丸のダイオードです。

キーソケットを交換して改善しないということで、基板の配線にダメージがある可能性もあります。
添付画像のオレンジと緑色どうし２点が導通しているかテスターでご確認いただけますでしょうか？
もしテスターをお持ちでない場合は思い切って同色どうしをエナメル線やゴム被覆銅線で接続してしまっても良いと思います。

お手数ですがご確認お願いいいたします。

![KB44_KeyFix3](https://github.com/Yowkees/keyball/assets/84209772/d3f1e60c-2976-4875-ada8-c2ed33a7736b)

**Reply by Leko** [reply]
@Yowkees ご確認ありがとうございます。オレンジと緑それぞれをテスターで同通チェックしてみた結果どちらも通じておりました。

**Reply by Leko** [reply]
![PXL_20230813_071558733~2](https://github.com/Yowkees/keyball/assets/1424963/624e03b2-553c-4af9-92a8-750865d48490)

ダイオードの極性も正しいと思います

**Comment by Yowkees** [comment, marked as answer]
あとは、赤丸で示したダイオードの半田付け不良のご確認をお願いします。
写真ですと半田が多く判断が難しいので、量を減らして基板のパッドに半田が馴染んでいるかご確認ください。
お手数ですがご対応お願いいたします。

**Reply by Leko** [reply]
ありがとうございます！赤丸の左側がパッドと馴染んでおらず不良が起こっておりました。
ハンダを付け直したところ無事直しました！

**Comment by Yowkees** [comment]
無事に直ったようで良かったです！安心しました。
ぜひ末永くKeyballシリーズをお楽しみいただければ幸いです。
以上、今後ともどうぞよろしくお願いいたします。
